packages/material-ui-codemod/README.md

```diff
@@ -0,0 +1,43 @@
+# Material-UI codemod
+
+This repository contains a collection of codemod scripts based for use with
+[JSCodeshift](https://github.com/facebook/jscodeshift) that help update Material-UI
+APIs.
+
+## Setup & Run
+
+- `npm install -g jscodeshift`
+- `npm install material-ui-codemod`
+- `jscodeshift -t <codemod-script> <path>`
+- Use the `-d` option for a dry-run and use `-p` to print the output
+  for comparison
+
+## Included Scripts
+
+### v0.15.0
+
+#### `import path`
+
+Updates the import paths for the new location of the components.
+Material-UI v0.15.0 is reorganizing the folder distribution of the project.
+Use this codemod to easily ugrade. The diff should look like this:
+
+```diff
+-import FlatButton from 'material-ui/src/flat-button';
+-import RaisedButton from 'material-ui/lib/raised-button';
++import FlatButton from 'material-ui/src/FlatButton';
++import RaisedButton from 'material-ui/lib/RaisedButton';
```

Comment:
I think so , yes. We should publish the next release as we intend the import paths to be in 0.15.0, so the first time any devs use this codemod, it will be for flattened imports.